enterprise/backend/src/metabase_enterprise/database_routing/middleware.clj

```diff
@@ -33,14 +34,15 @@
 (defenterprise attach-destination-db-middleware
   "Pre-processing middleware. Calculates the destination database that should be used for this query, e.g. for caching
   purposes. Does not make any changes to the query besides (possibly) adding a `:destination-database/id` key."
-  :feature :database-routing
```

Comment:
You can't be removing token feature checks and then not at least putting a note about why it doesn't do the feature check. Otherwise someone else is just going to look at this code that looks wrong and fix it